### DISCUSSION TITLE
Foggiest Clue Pt. II

### DISCUSSION THREAD

**Discussion by jasonjgardner** [jasonjgardner/jg-rtx#67] [Feedback]
> ![Minecraft screenshot](https://user-images.githubusercontent.com/1903667/182059019-cd46ea5d-5d74-43e4-a026-35e8bd92e5c1.png)
> Glass variety pack window options

**Pre-release:** Packs have not yet been tested. Release notes coming soon!

<hr /><em>This discussion was created from the release <a href='https://github.com/jasonjgardner/jg-rtx/releases/tag/v0.27.0'>Foggiest Clue Pt. II</a>.</em>

**Comment by ABUCKY0** [comment]
[https://learn.microsoft.com/en-us/minecraft/creator/reference/content/fogsreference/examples/fogsintroduction](https://learn.microsoft.com/en-us/minecraft/creator/reference/content/fogsreference/examples/fogsintroduction)
I believe this could help you with making your fogs that you want.
